docs/UserGuide.adoc

```diff
@@ -227,6 +227,35 @@ Format: `exit`
 Address book data are saved in the hard disk automatically after any command that changes the data. +
 There is no need to save manually.
 
+== Features(Coming in v2.0)
+
+=== Add relationship
+
+Enters the indices of the two people in between whom the user wants to add an relationship +
+as seen in the last shown list. +
+Format: `Add Relationship NAME1 NAME2`
+
+=== Add named relationship between people
+
+Enters the indices of the two people in between whom the user wants to add an relationship +
+as seen in the last shown list and the name of the relationship. +
+If the relationship between these two people is not in the address book, one new relationship will be created upon +
+entering this command and given a name. +
+The order of the names of people entered does not matter.
+Format: `Add Named Relationship NAME1 NAME 2 NAMEOFRELATIONSHIP`
+
+=== Add confidence estimate for an attribute of a person
+
+Enters the index of the person as shown in the last seen list, the name of the attribute +
+and the estimate (the estimate >= 0 and <= 1). +
+Format: `Add CEstimate INDEX NAMEOFATTRIBUTE ESTIMATE`
+
+=== Delete confidence estimate for an attribute of a person
+
+Enters the index of the person as shown in the last seen list and the name of the attribute +
+of which the confidence estimate is to be deleted. +
+Format: `Delete CEstimate INDEX NAMEOFATTRIBUTE`
+
 == FAQ
```

Comment:
In general, for all 4 commands introduced above, it would be good if you can add in some examples of the usages, particularly since the commands are slightly complicated and require several arguments